iosApp/iosApp/StarsView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TribeView.swift
+//  iosApp
+//
+//  Created by Jose Mateo on 1/23/22.
+//  Copyright © 2022 orgName. All rights reserved.
```

Comment:
@andymiller813 doesn't think we need the copyright line, I've been excluding them 🤷

Reply:
I'll remove it and tune my XC settings